Sources/prefrence/ALApplozicSettings.m

```diff
@@ -1989,5 +1989,26 @@ + (NSString *)getSelectedLanguageForSpeechToText {
     return [userDefaults valueForKey:KM_SELECTED_LANGUAGE_FOR_SPEECH_TO_TEXT];
 }
 
++ (void)setDefaultOverrideuploadUrl:(NSString *)url {
+    NSUserDefaults *userDefaults = [ALApplozicSettings getUserDefaults];
+    [userDefaults setValue:url forKey:KM_DEFAULT_UPLOAD_OVERRIDE_URL];
```

Comment:
`KM_DEFAULT_UPLOAD_URL` is sufficient, no? Same for headers.

Reply:
changed